repo.xml

```diff
@@ -14,6 +14,11 @@
     <permissions user="packageservice" password="packageservice" group="packageservice" mode="rwxrwxr-x"/>
 
     <changelog>
+        <change version="1.3.14">
+            <ul xmlns="http://www.w3.org/1999/xhtml">
+                <li>Load packages over a secure connection by default<a href="https://github.com/eXist-db/existdb-packageservice/pull/24">#24</a></li>
```

Comment:
I think ` - ` is needed between the description and the link

Reply:
OK, I'll add it